modules/external-api/src/main/java/org/opencastproject/external/endpoint/SeriesEndpoint.java

```diff
@@ -933,14 +935,22 @@ protected MetadataList deserializeMetadataList(String json) throws ParseExceptio
   @Path("{seriesId}/acl")
   @RestQuery(name = "updateseriesacl", description = "Updates a series' access policy.", returnDescription = "", pathParameters = {
           @RestParameter(name = "seriesId", description = "The series id", isRequired = true, type = STRING) }, restParameters = {
-                  @RestParameter(name = "acl", isRequired = true, description = "Access policy", type = STRING) }, reponses = {
+                  @RestParameter(name = "acl", isRequired = true, description = "Access policy", type = STRING),
+                  @RestParameter(name = "override", isRequired = false, defaultValue = "false", description = "If true the series ACL will take precedence over any existing episode ACL", type = STRING)}, reponses = {
```

Comment:
Should be of type BOOLEAN (requires import static org.opencastproject.util.doc.rest.RestParameter.Type.BOOLEAN;) so that the REST docs display a checkbox.

Reply:
Thanks, fixed!